pkg/encryption/encryption.go

```diff
@@ -185,3 +185,10 @@ func GenerateRandomKey(l int) Key {
 	}
 	return key
 }
+
+func min(a, b int) int {
```

Comment:
this function should already be built into go1.21

Reply:
ah yeah, you are right since then I updated :)